package.json

```diff
@@ -21,10 +21,11 @@
   "devDependencies": {
     "chloride": "^2.2.8",
     "pull-pushable": "^2.2.0",
+    "tap-spec": "^5.0.0",
     "tape": "^5.0.1"
   },
   "scripts": {
-    "test": "set -e; for t in test/*.js; do node $t; done"
+    "test": "tape test/*.js | tap-spec"
```

Comment:
:spiral_notepad: Note: I think this should be `tape 'test/*.js'` if we want it to run tests on Windows.

Reply:
They do run on Windows :-)